src/monopoly/monopoly_player.js

```diff
@@ -75,13 +71,15 @@ MonopolyPlayer.prototype.owesRent = function(property) {
 };
 
 MonopolyPlayer.prototype.owns = function(property) {
-  return property.owner === this;
+  return property.owner && property.owner.id === this.id;
```

Comment:
Understood. I’ll keep this in mind in the future.

I think this baby is looking pretty good to merge.

> On Feb 23, 2016, at 11:04 PM, andrewjmeier <notifications@github.com> wrote:
> 
> In src/monopoly/monopoly_player.js <https://github.com/andrewjmeier/TableTop.JS/pull/91#discussion_r53891438>:
> 
> > @@ -75,13 +71,15 @@ MonopolyPlayer.prototype.owesRent = function(property) {
> >  };
> >  
> >  MonopolyPlayer.prototype.owns = function(property) {
> > -  return property.owner === this;
> > +  return property.owner && property.owner.id === this.id;
> No because when the objects get recreated we're not creating a player object at the top level and passing it all the way in. We're passing the JSON in from the top and the property say "oh I need to build a player with this data" Personally I don't think a property should have a reference to a player since we talked a bit about it being like a token. Should only be player -> property
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/andrewjmeier/TableTop.JS/pull/91/files#r53891438>.
>